qa/L0_model_config/compare_status.py

```diff
@@ -26,49 +26,60 @@
 
 import argparse
 import os
-import os.path
 import sys
-from tensorrtserver.api import *
-import tensorrtserver.api.server_status_pb2 as server_status
-import tensorrtserver.api.model_config_pb2 as model_config
+import json
+import tritonhttpclient.core as httpclient
```

Comment:
These imports don't exist anymore, make sure you rebase to the top of v2test and you will see that this test fails.